cmd/voucher_subscriber/README.md

```diff
@@ -0,0 +1,64 @@
+# Voucher Subscriber
+
+- [Installation](#installation)
+- [Configuration](#configuration)
+- [Usage](#usage)
+
+## Installation
+
+Install `voucher_subscriber` by running:
+
+```shell
+$ go get -u github.com/grafeas/voucher/cmd/voucher_subscriber
+```
+
+This will download and install the voucher subscriber binary into `$GOPATH/bin` directory.
+
+## Configuration
+
+See the [Tutorial](TUTORIAL.md) for more thorough setup instructions.
+
+An example configuration file can be found in the [config directory](../../config/config.toml).
+
+The configuration can be written as a toml, json, or yaml file, and you can specify the path to the configuration file using the `-c` flag.
+
+All of the configuration options for the Voucher Subscriber is the same as the [Voucher Server](../voucher_server/README.md#configuration)
+
+## Usage
+
+You can run Voucher in server mode by launching `voucher_subscriber`, using the following syntax:
+
+```shell
+$ voucher_subscriber [--project <project> --subscription <subscription>]
+```
+
+`voucher_subscriber` supports the following flags:
+
+| Flag        | Short Flag       | Description                                                                |
+| :--------   | :--------------- | :------------------------------------------------------------------------- |
+| `--project`  | `-p`            | The GCP project to be used.                                                |
+| `--subscription` | `-s`        | The subscription that contains messages.                                   |
+| `--timeout` |                  | The number of seconds to spend checking an image, before failing.          |
+
+For example:
+
+```shell
+$ voucher_subscriber --project my-project --subscription my-subscription
+```
+
+This would launch the subscriber listening to `my-subscription` in `my-project`.
+
+The subscriber expects messages with the following format:
+
+```json
+{
+  "action":"INSERT",
```

Comment:
question: does this match the message format of Container Analysis topic?

Reply:
This matches the format used by GCR, sorry I forgot we were looking at containersnalysis pubsub!

Reply:
yeah sorry, I was under the impression we would be using the topic from the GCR topic! however the good thing here is that we can easily add options to listen to various types of default topics that expect different types of payloads; the end logic is the same for all of them as we want to take an image and start vouching it